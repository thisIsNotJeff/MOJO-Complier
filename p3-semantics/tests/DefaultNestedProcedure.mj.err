 line 3, column 5: error: default is a nested procedure (bar)
