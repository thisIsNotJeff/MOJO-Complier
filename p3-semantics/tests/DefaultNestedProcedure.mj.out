BEGIN _BUILTIN
_BUILTIN.int=int
_BUILTIN.bool=bool
_BUILTIN.false:bool=false
_BUILTIN.true:bool=true
_BUILTIN.Null=^null
_BUILTIN.nil:Null=0
_BUILTIN.Refany=^null
_BUILTIN.Address=^null
_BUILTIN.Root=null
object {
}
_BUILTIN.Text=^null
_BUILTIN.First:(...)
_BUILTIN.Last:(...)
_BUILTIN.New:(...)
_BUILTIN.Number:(...)
BEGIN DefaultNestedProcedure
BEGIN DefaultNestedProcedure.1
DefaultNestedProcedure.1.T=Root
object {
0: bar:()=DefaultNestedProcedure.1.Bar:(VALUE t:T)
}
DefaultNestedProcedure.1.Bar:(VALUE t:T)
BEGIN DefaultNestedProcedure.1.Bar
DefaultNestedProcedure.1.Bar.t:T
END DefaultNestedProcedure.1.Bar
END DefaultNestedProcedure.1
END DefaultNestedProcedure
END _BUILTIN
